client/components/CandidateReview/CandidateTestPlanRun/InstructionsRenderer.jsx

```diff
@@ -169,19 +170,153 @@ const InstructionsRenderer = ({
         commandsContent
     );
 
-    const assertions = [...pageContent.instructions.assertions.assertions];
-    const assertionsContent = parseListContent(assertions);
-
+    headingLevel = 5;
```

Comment:
```suggestion

```

Noticed this after going to the CandidateTestPlanRun page which also uses this component and realized the scenarioTitle heading component is wrong. 

You shouldn't override what's being passed into this component. Instead pass in the `headingLevel` prop as `5` where `InstructionsRenderer` is being called in the `TestReview` component.